scripts/pytests.nix

```diff
@@ -0,0 +1,45 @@
+# This is for development convenience, to make setting up and running pytests straightforward.
+# This is not a final product, so feel free to edit manually to adjust tests running or do whatever is convenient
+#   for you right now.
+#
+# Build binaries: `cargo build --release --bins`
+# Run from the repo root: `nix-shell ./scripts/pytests.nix`
+# That's it
+with import <nixpkgs> {};
+  pkgs.mkShell {
+    buildInputs = [
+      poetry
+      python313Packages.tomli-w
+
+      cowsay
+    ];
+
+    BIN_IROHAD = "target/release/irohad";
+    BIN_IROHA = "target/release/iroha";
+    BIN_KAGAMI = "target/release/kagami";
```

Comment:
What’s the rationale for switching to release builds? Is it to align with integration tests that use `cargo install`? However, unless we’re testing non‑functional requirements, wouldn’t debug builds suffice?

Reply:
Yes, debug builds would totally suffice.

I made it to align with the integration tests. Which aren't, btw, using `cargo install` anymore; they rely on `target/release` bins too.

Reply:
It might be helpful to update the `README.md` or `CONTRIBUTING.md` with how to run the integration tests.

Reply:
Created: https://github.com/hyperledger-iroha/iroha/issues/5505